openmc/mesh.py

```diff
@@ -1354,6 +1378,7 @@ def __init__(self, mesh_id=None, name=''):
         self._r_grid = None
         self._theta_grid = [0, pi]
         self._phi_grid = [0, 2*pi]
+        self._origin = (0., 0., 0.)
```

Comment:
Similar fix to the one above using the property setter.
```suggestion
        self.origin = (0., 0., 0.)
```